store/modules/heatmap/mutations.js

```diff
@@ -33,4 +33,9 @@ export default {
   set_heatmap_options_gradient(state, new_value) {
     state.heatmap_options_gradient = new_value;
   },
+  set_metric_data(state, new_value) {
```

Comment:
this is for testing right now, we need to design a better way to handle the streaming data, probably in it's own store other components read from with a compute method